sdk/keyvault/azkeys/client.go

```diff
@@ -237,26 +240,28 @@ func (c *Client) CreateECKey(ctx context.Context, name string, options *CreateEC
 	return createECKeyResponseFromGenerated(resp), nil
 }
 
-// CreateOctKeyOptions contains the optional parameters for the Client.CreateOCTKey method
+// CreateOctKeyOptions contains optional parameters for CreateOctKey
 type CreateOctKeyOptions struct {
-	// Hardware Protected OCT Key
+	// HardwareProtected determines whether the key is is created in a hardware security module (HSM).
 	HardwareProtected *bool
 
-	// The key size in bits. For example: 2048, 3072, or 4096 for RSA.
+	// Size is the key size in bits.
```

Comment:
The previous comments suggest that the value _should_ be a power of two and that there _might_ be a limit on acceptable values.  Are either of these true?

Reply:
Yes. I'll list the legal values.